src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -990,8 +996,15 @@ export class PostgresStorageAdapter implements StorageAdapter {
       }
     });
     await conn.tx('set-indexes-with-schema-format', async t => {
-      if (insertedIndexes.length > 0) {
-        await self.createIndexes(className, insertedIndexes, t);
+      try {
+        if (insertedIndexes.length > 0) {
+          await self.createIndexes(className, insertedIndexes, t);
+        }
+      } catch (e) {
+        const columnDoesNotExistError = e.errors?.[0]?.code === '42703';
```

Comment:
Optional chaining is available from Node 14; Parse Server still supports Node 12.

Reply:
You can use optional changing now, Parse Server 6 supports Node >=14